terraform/modules/iam-users/developers.tf

```diff
@@ -93,7 +93,7 @@ resource "aws_iam_group_policy_attachment" "dev_s3_only_iam_manage_account" {
   policy_arn = "${var.iam_manage_account_policy_arn}"
 }
 
-resource "aws_iam_policy" "dev_uploads_s3_access" {
+resource "aws_iam_policy" "dev_s3_access" {
   name = "devUploadsS3Access"
 
   policy = <<POLICY
```

Comment:
Is the plan to move this inline policy to its own `aws_iam_policy_document` at some point as well?

Reply:
Yeah, wanted to avoid the PR noise. I actually put in a ticket on the tech debt board this morning to have a look at some of this stuff. 
Problem is converting this one now means I should probably convert the jenkins one too, and that's a beast.